Gruntfile.js

```diff
@@ -334,7 +334,8 @@ module.exports = function (grunt) {
           'app/scripts/services/simplesliderservice.js',
           'app/scripts/directives/simple-slider.js'
         ],
-        dest: 'dist/angular-simple-slider.js' },
+        dest: 'dist/angular-simple-slider.js'
+      },
```

Comment:
hmmm, mine doesn't complain, maybe I need to specify some missing setting on `.jshintrc`.

Anyways, it's not related to the changes so it's better to revert it for the sake of keeping the PR focused only on the things it's meant to fix :)